src/transformer/descriptor/module/module.ts

```diff
@@ -134,7 +134,8 @@ export function GetPropertiesFromSourceFileOrModuleDeclaration(
         }
 
         return TypescriptCreator.createPropertySignature(
-          d.originalDeclaration.name as ts.Identifier,
+          (d.originalDeclaration.name as ts.Identifier) ||
+            ts.createIdentifier('default'),
```

Comment:
Do you still need the casting?

Reply:
yes, apparently `name` can be also a "DeclarationName" that doesn't match "Identifier"